00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+
+1. **Split into groups**: The class will be divided into groups using the breakout sections feature of Zoom, these assignments are random.
+   1. Breakout Room 1:  Aardvark
+   2. Breakout Room 2:  Tarsier
+   3. Breakout Room 2:  Gecko
+   4. Breakout Room 2:  Sloth
+   5. Breakout Room 2:  Lionfish
+   6. Breakout Room 2:  Pangolin
+   7. Breakout Room 2:  Rhino
+   8. Breakout Room 2:  Curassow
+2. **Fork the activity repo**: One person in your group should fork the carnival-recipes repo and then add everyone else as a collaborator.
+3. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+4. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are listed in the carnival-recipes repo. If your group has fewer than six people someone may need to take multiple scraps.
+5. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and then add and commit your changes to the recipe file. After completing this step each person in your group should have **different** content within the recipe file in their local repository.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- If you're still unsure about what a merge conflict is, or how to resolve one please review the [Merge Conflicts curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/git-merge-conflicts.md).
+- You can review the [slides](https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit?usp=sharing) and [example walkthrough video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ec7dbf60-e59b-4317-a4a5-abbe002eb7ea) if you are unsure about how to complete the activity.
+- The end result should be a real, usable recipe. The original recipe files were all complete works from various recipe websites online.
+- If you find that the result of one of your merges is missing critical ingredients or any numbered steps, you should go back and double check that you didn't lose any lines while resolving a merge conflict.
+- Your recipe scraps may have duplicate lines. We've added these in to more accurately portray the full range of potential merge conflicts. **Important**: you should get rid of any duplicate lines while merging.
+- There's no "right" way to do this activity, all of the strategies listed below are valid ways to build the recipe file.
+- Have fun!
+
+#### Recipe building process
```

Comment:
To be honest, I find the nested list of lists of lists disorienting and hard to follow. 
I am inclined to suggest we just reuse the instructions from the [original README](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/exercises/merge-conflict.md) and count on the instructors' support to give further clarity.

Reply:
The original doesn't use branches.